velox/functions/sparksql/String.cpp

```diff
@@ -103,6 +103,286 @@ class Length : public exec::VectorFunction {
   }
 };
 
+class ConcatWs : public exec::VectorFunction {
+ public:
+  explicit ConcatWs(const std::optional<std::string>& separator)
```

Comment:
Can this function be implemented as a simple function? If not, let's move it into a separate file.